pkg/client/restclient/client.go

```diff
@@ -103,8 +104,9 @@ func NewRESTClient(baseURL *url.URL, versionedAPIPath string, config ContentConf
 	} else if rateLimiter != nil {
 		throttle = rateLimiter
 	}
+	glog.V(4).Infof("Rest client initialized with following urls %v", hosts)
 	return &RESTClient{
-		base:             &base,
+		urlProvider:      &RoundRobinProvider{urls: hosts},
```

Comment:
and use pointer when Request object will be created? But then on each new request we will try to connect to potentially broken api-server